src/scenes/Partners/Edit/EditPartner.tsx

```diff
@@ -128,6 +133,25 @@ const fieldMapping = {
   'partner.countries': ({ props }) => (
     <LocationField {...props} label="Countries" multiple variant="outlined" />
   ),
+  'partner.languageOfReportingId': ({ props }) => (
+    <LanguageField
+      {...props}
+      label="Language of Reporting"
+      CreateDialogForm={undefined}
```

Comment:
I would not try to forcibly disable language creation here. This is a deviation from the standard practice of the app. It is also not necessary because only admins can create languages, so only admins would have the create language option anyways.